test/functional/fixtures/regression/gh-993/testcafe-fixtures/index.test.js

```diff
@@ -0,0 +1,20 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `t.pressKey on angular2 element`
```

Comment:
oh, I missed this.

Something like `Check key and keyIdentifier event properties`